tests/test_transforms/test_inference/conftest.py

```diff
@@ -169,21 +169,21 @@ def ts_to_resample() -> TSDataset:
 def ts_to_resample_int_timestamp() -> TSDataset:
     df_1 = pd.DataFrame(
         {
-            "timestamp": np.arange(120),
+            "timestamp": np.arange(24, 144),
```

Comment:
Why did you change it?

Reply:
We changed it to check that resampling uses `.loc`.